parser.y

```diff
@@ -6324,84 +6327,105 @@ TableOptimizerHintList:
 	}
 
 TableOptimizerHintOpt:
-	index '(' HintTableAndIndexList ')'
+	index '(' QueryBlockOpt HintTable IndexNameList ')'
```

Comment:
Do we have other way to do this? `INDEX(@sel_1 t idx1, idx2, idx3)` looks weird. Because `IndexNameList` here can not be empty, we can split by comma.

Reply:
It can be empty now.